demo/angularjs/components/tabs/doc.md

```diff
@@ -0,0 +1,15 @@
+# Tabs
+
+**A Tab is a sub-navigation that show content selected.**
```

Comment:
Plural subject -> `Tabs`
Can the tab be the main navigation? Maybe you could say `Tabs are navigation elements`
How is the content `selected`? I think what you are talking about is the `active tab`. Can you try and rewrite it around this term?

Reply:
I've replaced by "Tabs are navigation elements that able users to navigate between related sections of content." What do you think? @lionelmoi

Reply:
`that able users` is not very common :)

"Tabs are navigation elements that regroup related sections of content for users to navigate."

Reply:
Thanks!